web/frontend/src/components/gamepad/index.svelte

```diff
@@ -0,0 +1,328 @@
+<script lang='ts'>
+
+import { grpc } from '@improbable-eng/grpc-web';
+import { onMount, onDestroy } from 'svelte';
+import { Timestamp } from 'google-protobuf/google/protobuf/timestamp_pb';
+import { ConnectionClosedError } from '@viamrobotics/rpc';
+import {
+    Client,
```

Comment:
Should be 2 space indents, I guess linting tools aren't working there yet, I'll make sure to set that up as a next step